src/NHibernate/Linq/ReWriters/RemoveUnnecessaryBodyOperators.cs

```diff
@@ -36,10 +36,9 @@ public override void VisitResultOperator(ResultOperatorBase resultOperator, Quer
 			}
 			if (resultOperator is AnyResultOperator)
 			{
-				Array.ForEach(queryModel.ResultOperators.OfType<FetchOneRequest>().ToArray(), op => queryModel.ResultOperators.Remove(op));
-				Array.ForEach(queryModel.ResultOperators.OfType<FetchManyRequest>().ToArray(), op => queryModel.ResultOperators.Remove(op));
+				ResultOperatorRemover.Remove(queryModel, x=>x is FetchRequestBase);
```

Comment:
Lacking spaces around  `=>`. This may cause unrelated changes later on we working again on that code.